demo/Vagrantfile

```diff
@@ -22,15 +24,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.synced_folder ".", "/vagrant", disabled: true
 
-  config.vm.provision "shell", privileged: false, inline: <<SCRIPT
-    ## Uncomment these lines to update flynn before starting
-    # sudo apt-get update -qq
-    # sudo apt-get install -y flynn-host
-    # sudo flynn-host download /etc/flynn/version.json
-
-    sudo start flynn-host
+  config.vm.provider "virtualbox" do |v|
+    v.gui = true
```

Comment:
FYI the contents of `/etc/issue` are printed before the login prompt, see [`login(5)`](http://linux.die.net/man/5/issue)